deploy/tasks/assets.cap

```diff
@@ -1,3 +1,6 @@
+require 'rubygems'
+require 'json'
```

Comment:
I guess this also needs to be added to the `CHANGELOG`, right? Like `bundle add json`.